cmd/timestamp-cli/app/verify.go

```diff
@@ -85,6 +97,62 @@ func runVerify() (interface{}, error) {
 		return nil, err
 	}
 
+	opts, err := verification.NewVerificationOpts(ts, artifact, pemBytes)
```

Comment:
@haydentherapper curerntly all verification is handled in a single function `VerifyTimestampResponse` exported from the `verification` package. Here I add new verification functions separately. Do you ahve thoughts on whether all new verification functions should be called from `VerifyTimestampResponse` or if they should be called directly from here?

Reply:
I think it'd be best to have a single exported verification function that performs all other verifications. If you think there's value in exporting the sub-verification functions, maybe we put them under an internal package?

Reply:
Happy to use a single exported function, I'll make that change.